### ISSUE TITLE
NiceHash Miner v2.0.1.8 stuck at Starting Excavator Server

### ISSUE THREAD

**Issue by PredatorAaron** [nicehash/NiceHashMiner-Archived#1427]
I just update the NiceHash Miner an hour ago and after finish updating I wanted to open my app it just stucks at the "Starting Excavator Server" I've been waiting for an hour already, does anyone encounter this problem after updating to v2.0.1.8?

**Comment by MITSTE** [comment]
Same issue here, I received notification that an update was available and after I installed it, now just stays stuck at excavator sever phase, I uninstalled and reinstalled but to no avail. Luckly I run two mining programs at once so I just switched all GPU power to the other program, I'm not losing money because their program doesn't work well because it in beta.

**Comment by PredatorAaron** [comment]
What's the other mining program are you using?

**Comment by PredatorAaron** [comment]
What's the other mining program are you using?

**Comment by MITSTE** [comment]
Minergate v.7.2
Nicehash won't touch my i7 8 core processor for whatever reason so I merge mine on the other program with it. So I mine 3 crypto's at once when everything is working fine.

**Comment by PredatorAaron** [comment]
Thanks

**Comment by nojuan95** [comment]
has this problem been fixed for you guys because now min just freezes on start up

**Comment by MITSTE** [comment]
Nope, I've been doing research since this post and haven't gotten anywhere playing around with exclusions in the antivirus and running the program as administrator. I even rolled back to the previous version and now that no longer works either. I had just submitted a ticket to Nicehash, my balance information also disappeared from their website so I might have just lost my money as well.

**Comment by nojuan95** [comment]
how does minergate work anyone know

**Comment by MITSTE** [comment]
Works better than Nicehash and you at least have options in which crypto you want to mine and be paid in. But you have to create wallets for whatever crypto you mine or use their endorsed partners exchange.

**Comment by nojuan95** [comment]
what is a good one for wallet for monero

**Comment by MITSTE** [comment]
Go to Monero's website they have one for download.

**Comment by nojuan95** [comment]
ok cool

**Comment by nojuan95** [comment]
im new to all this so thanx for the help

**Comment by JackG123** [comment]
Hi guys, someone has helped me fix this issue on the nicehash facebook page, you have to get the openCL.dll file and copy it in to your excavator folder

**Comment by JackG123** [comment]
The excavator folder is normally on your computer at c:\users\(yourusername)\AppData\Roaming\nhm2\bin\excavator

**Comment by nojuan95** [comment]
where do we get that file

**Comment by JackG123** [comment]
you might already have it on your computer, hit start button and type in OpenCL.dll it will do a search for it

**Comment by JackG123** [comment]
[OpenCL.zip](https://github.com/nicehash/NiceHashMiner/files/1636452/OpenCL.zip)

**Comment by MITSTE** [comment]
Attempting.

**Comment by nojuan95** [comment]
cant find the file location

**Comment by JackG123** [comment]
nojuan95 when you go into the bin folder is there no excavator folder?

**Comment by nojuan95** [comment]
bin folder for what

**Comment by JackG123** [comment]
The excavator folder is normally on your computer at c:\users(yourusername)\AppData\Roaming\nhm2\bin\excavator

**Comment by MITSTE** [comment]
Still stuck. But did exactly as you had posted. Another issue?

**Comment by nojuan95** [comment]
c:\users(yourusername)\AppData\Roaming\nhm2\bin\excavator where do i put this in at

**Comment by JackG123** [comment]
nojuan95 you need to copy the file in the zip i posted the OpenCL.dll file and paste it in to the excavator folder

**Comment by JackG123** [comment]
MITSTE did you close the nice hash miner and open it again

**Comment by MITSTE** [comment]
JackG123
Any solutions?

**Comment by nojuan95** [comment]
there is no excavator folder

**Comment by nojuan95** [comment]
there is no excavator  folder

**Comment by nojuan95** [comment]
so i got up to the app data folder and there is no roaming folder

**Comment by JackG123** [comment]
nojuan95 did you have all the other folders up till that

**Comment by JackG123** [comment]
MISTSTE did you close the nice hash miner and open it again?

**Comment by nojuan95** [comment]
what do you mean

**Comment by JackG123** [comment]
nojuan95 when you get to app data what folders are there?

**Comment by nojuan95** [comment]
so i got to the bin folder now but there is no excavator folder

**Comment by JackG123** [comment]
sorry in bin the folder is called excavator_server

**Comment by nojuan95** [comment]
is it a zip file

**Comment by JackG123** [comment]
if you dont have that folder, get it from here 

https://github.com/nicehash/excavator

**Comment by MITSTE** [comment]
JackG123
Yep. Still nothing.

**Comment by JackG123** [comment]
sorry get it from here

https://github.com/nicehash/excavator/releases

**Comment by JackG123** [comment]
MITSTE you could try this too get the latest excavator

**Comment by nojuan95** [comment]
so what do i do with that file

**Comment by nojuan95** [comment]
says i cant run the exe file

**Comment by MITSTE** [comment]
Waiting to sell out of RTN puts. Be back shortly.

**Comment by nojuan95** [comment]
ok

**Comment by JackG123** [comment]
create a new folder in the bin folder called excavator_server then paste it all in to that folder

**Comment by nojuan95** [comment]
ok and after i do that

**Comment by JackG123** [comment]
after you do that copy and paste the opencl.dll file in to the excavator folder then open the nicehash miner and it should work

**Comment by nojuan95** [comment]
it keeps freezing at the downloading xmr stack cpu miner

**Comment by MITSTE** [comment]
JackG123
Still nothing. Damn.

**Comment by nojuan95** [comment]
it just keeps freezing

**Comment by JackG123** [comment]
Only other thing I could suggest guys is to uninstall nicehashminer, restart your computer, install nicehashminer, make sure you have the excavator folder or get it from the link above, then make sure you have the opencl.dll file in that folder, then open nicehashminer

**Comment by nojuan95** [comment]
ill try that

**Comment by MITSTE** [comment]
JackG123
Just was working on that right before you posted. Will see if that works hopefully.

**Comment by nojuan95** [comment]
Its starting up alittle better for me now just loading the excavator server

**Comment by nojuan95** [comment]
Its starting up alittle better for me now just loading the excavator server

**Comment by nojuan95** [comment]
Now its just stuck on the starting excavator server

**Comment by MITSTE** [comment]
My computer crashed. So will start the process over again.

**Comment by nojuan95** [comment]
So when i go to the excavator server file after trying to start nice hash there is no excavator.exe file

**Comment by nojuan95** [comment]
Excavator by nice hash is cirrently offline

**Comment by MITSTE** [comment]
Doesn't get anymore f---k'ed up then that computer just shut off completely and had to turn it back on again and now the program loads up perfectly. The program is running the benchmark so we will see short what happens.

**Comment by nojuan95** [comment]
Ok

**Comment by nojuan95** [comment]
Mine is doing the same now its at the benchmark stage

**Comment by MITSTE** [comment]
nojuan95
Did it shutoff to?
I'm back-up N mining and have no idea what had happened in between everything. Let me know if yours works out.

**Comment by rf9661** [comment]
Hi!

I used to have this problem until I noticed it was my Anti-Virus/Firewall. I disabled the McAfee 'Real Time Scanning' and it fixed the issue where NiceHash would just stay stuck at the excavator part.

**Comment by MITSTE** [comment]
rf9661
It wasn't the antivirus as I made exclusions for these programs (I have two mining programs), it had something to do with Nicehash running twice simultaneously, I had found out watching task manager and saw two Nicehash programs running at once cancelling out the other which later cause my computer to shutoff for safety as I had multiple other programs also running. When I had started the computer back up it tried running Nicehash twice again before I hit end task to both before reopening the program and getting it to respond, functioning as intended.

But thanks for the advice as I know other people including myself had issues with Windows 10 and the antivirus disabling and deleteing these (mining) programs.

**Comment by PredatorAaron** [comment]
I haven't tried this method yet but some people said it actually works, this link: https://github.com/nicehash/excavator/issues/120

**Comment by nojuan95** [comment]
Mine is to

**Comment by nojuan95** [comment]
I had to approve the program with my antivirus software

**Comment by orbeleany** [comment]
I found the solution First you have to Exclude the entire nicehash Folder from all your anti viruses and then restore the files that where detained by the antiviruses and relaunch the program and it should work.

**Comment by wagnerbenh** [comment]
excavator.exe is getting quarantined by Malwarebytes for me. Has this been compromised?

![image](https://user-images.githubusercontent.com/1786494/35080986-3e2c6a2a-fbdf-11e7-8259-056aacc99e8f.png)

**Comment by orbeleany** [comment]
Yeah just excluded from malwarebites and bit defender if you have it and
restored any file and it should work fine

**Comment by Quantdax** [comment]
I have a problem on WIN7. When i install program, it didnt create \BIN folder, so i dont have excavator folder. Did annyone had this kind of problem? I have 2x GTX 1080.

Tnx for anny suggestions

**Comment by nojuan95** [comment]
When u open the program did u get a warning from your anti virus

**Comment by Quantdax** [comment]
No. I dont have anny AV program. Windows Firewall is disabled.

**Comment by nojuan95** [comment]
So u just didnt get the excavator file

**Comment by Quantdax** [comment]
I didnt, no. But thats not the problem. Problem is that i dont have BIN folder in which i could copy EXCAVATOR folder.

**Comment by nojuan95** [comment]
o thats odd i havnt seen that problem before and are u searching in the right spot.

**Comment by nojuan95** [comment]
c:\users(yourusername)\AppData\Roaming\nhm2\bin\excavator 

so your telling me u cant do this.

**Comment by Quantdax** [comment]
Yap. 
c:\users(yourusername)\AppData\Roaming\nhm2
I have it like this. No BIN folder in which i could put excavator folder. Or if BIN sgows up, maybe it will be allready inside. I reinstalled couple of times. Simply no BIN folder. Program starts normally, recognizes 2x GTX 1080, but i cant benchmark

**Comment by nojuan95** [comment]
thats weird

**Comment by nojuan95** [comment]
i honestly don't know how to help you with this one i have never encountered this problem

**Comment by nojuan95** [comment]
i wonder if you could just make the bin folder yourself and just put the file in there .

**Comment by Quantdax** [comment]
It says missing mining file in benchmark under speed tab. I would really appriciate anny help. 
One other question. Is NHM2 better in minning over v 1.8? If not, no point stugelling with this install

**Comment by nojuan95** [comment]
thats what i am currently using

**Comment by Quantdax** [comment]
I tried just create BIN but it didnt help.

**Comment by nojuan95** [comment]
that is strange

**Comment by Quantdax** [comment]
So, i installed WIN10. Its stuck on STARTING EXCAVATOR SERVER.
Problem is that whn i start NHM2, it downloads folder called excavator_cuda9.1 to BIN folder.
Anny attempt do C/P EXCAVATOR_SERVER folder in BIN folder is ok, but whn NHM2 start, he creates and uses this CUDA folder...i have no idea what to do. Anny suggestions?

**Comment by FredrikLangen** [comment]
I can seem to find the bin folder in nhm2. i have downloaded excavator and OpenCL.zip but i cant make changes inside the nhm2 folder, like i cant add things? help

**Comment by FredrikLangen** [comment]
I can only see Cache and Logs and Cookies, but not bin

**Comment by Germanode** [comment]
I have the same problem since a few days ago. Antivirus is set to ignore all nicehash folders, including the one in AppData. I checked for OpenCL.dll and it's where it should be. Also I tried redownloading excavator files manually. Nothing... Then I reinstalled nicehash - still stuck on "Starting Excavator Server". However, if I open excavator.exe with a .bat file with custom settings, everything works and starts up fine. Little help, dev?

**Comment by orbeleany** [comment]
check your any anti viruses that you have and make sure they aren't
retaining any files and if so restore them

**Comment by Germanode** [comment]
AVG has all NiceHash folders as an exception and it put nothing in the quarantine - I've just checked. Windows defender was disabled while doing the reinstall. It still is disabled.

**Comment by orbeleany** [comment]
windows defender doesn't restrict it atleast not with me. But make sure if
you have other anti like bitdefender or malwarebytes restrict certain
files. My best bet would be to uninstall the anti viruses or desable it
completely before reinstalling it.

**Comment by Germanode** [comment]
Tried that just now with AVG completely disabled. Reinstalled. No luck. Still stuck at Starting Excavator server - whereas excavator.exe opens just fine with my .bat file. No admin privileges while opening.

**Comment by orbeleany** [comment]
try adding it to trusted files on your firewall

**Comment by Germanode** [comment]
Both Nicehash and excavator are set as allowed by the fiewall (AVG handles firewall). I've also tried with the fully disabled firewall.
